tests/phpunit/tests/cron.php

```diff
@@ -125,6 +124,30 @@ function test_wp_schedule_event_without_cron_option_does_not_throw_warning() {
 		$this->assertTrue( wp_schedule_event( $timestamp, 'daily', $hook ) );
 	}
 
+	/**
+	 * Tests that a call to wp_schedule_single_event() on a site without any scheduled events
+	 * does not result in the value "false" being added into the cron array.
+	 *
+	 * @ticket 53950
+	 *
+	 * @covers ::wp_schedule_single_event
+	 */
+	function test_wp_schedule_single_event_without_cron_option() {
+		delete_option( 'cron' );
+
+		// Verify that the cause of the error is in place.
+		$this->assertFalse( _get_cron_array(), '_get_cron_array() does not return false' );
+
+		$hook      = __FUNCTION__;
+		$timestamp = strtotime( '+10 minutes' );
+
+		// Add an event.
+		$this->assertTrue( wp_schedule_single_event( $timestamp, $hook ) );
+
+		// Verify that "false" is not a value in the final cron array.
+		$this->assertNotContains( false, get_option( 'cron' ) );
```

Comment:
Just realized that I forgot to add a `$message` parameter to these two assertions. @peterwilsoncc Shall I still add this ? Would make debugging easier if the test would even fail in the future.

Reply:
Good spot, may as well add them now rather than in a follow up commit. Thanks @jrfnl

Reply:
Updated the relevant commit and force pushed. No other changes.